CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Objects module.
+- New class to store face authentication result data (instead of proprieties in FaceAuthentication class).
```

Comment:
Shouldn't this entry be moved to the "Changed" and perhaps just simplifying the entry for example "Face authentication result data to its new class". If you find it makes sense, it does to me :)